src/tailwind.preset.js

```diff
@@ -25,5 +25,8 @@ export default {
       notice: '#fbc168',
       error: '#d93c3f',
     },
+    spacing: {
+      'touch-minimum': '44px', // Equivalent to spacing 11; minimum touch-target size
```

Comment:
Can you explain a bit more about how this is or will be used? I couldn't find a reference in the Tailwind docs.

Reply:
@robertknight This is extending the `spacing` theme configuration values to include a custom value: `touch-minimum`. This makes a full set of tailwind utility classes based on this token available (e.g. `w-touch-minimum` or `m-touch-minimum`).

It's an abstracting token so that we can use `w-touch-minimum` (or `h-touch-minimum`) instead of the equivalent `w-11` (Tailwind's default spacing scale is `1 unit = 4px` or more accurately `0.25rem`) to provide more context. The set of default values for spacing an be seen on [Tailwind's documentation for customization of spacing units](https://tailwindcss.com/docs/customizing-spacing). To put it another way: `touch-minimum` is our own invention; we're extending Tailwind's defaults with this Hypothesis-custom token.

I had already added this locally to LMS' tailwind configuration and [made use of it in the `StudentSelector` component](https://github.com/hypothesis/lms/blob/9c1e56cc3696506641a02dbb654bcb4f5cc33b17/lms/static/scripts/frontend_apps/components/StudentSelector.js#L82). This is me extracting it to the common preset used by all apps.

Is this still unclear in any way? I'm happy to make it more clear—I want this to make sense!

Reply:
This makes sense, thanks!